components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOService.java

```diff
@@ -226,6 +226,7 @@ public void doSingleLogout(String sessionId, String issuer) throws IdentityExcep
             String key = entry.getKey();
             SAMLSSOServiceProviderDO serviceProviderDO = entry.getValue();
 
+            // TODO : UI configuration to check for Front-Channel SLO for SPs and filter out.
```

Comment:
why TODO added?

Reply:
As this PR was sent to a feature branch this TODO was added. This is addressed later with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/208.

Reply:
if this is done shall we remove this

Reply:
This is completed and the todo is removed in #208